src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py

```diff
@@ -35,6 +35,7 @@
 helps['webapp config set'] = """
     type: command
     short-summary: create or update web app configurations.
+    long-summary: For Windows Node webapp, use appsetting of 'WEBSITE_NODE_DEFAULT_VERSION'. For Linux Node webapp, use '--linux-fx-version'
```

Comment:
The root cause is the server API is rather inconsistent as all other stack run time are still being supported through the site config. Let me know if there are better places for this; otherwise the comment can still stay as an important clarification